### ISSUE TITLE
Please do not move to the coc's scratch floating window

### ISSUE THREAD

**Issue by yutakatay** [mg979/vim-xtabline#11]
Thank you for your continuous support. This plugin is one of my favorite

---

The current xtabline moves to the floating window displayed by Coc.nvim.
I want to add an option to not move in it.

![Peek 2020-05-08 16-23](https://user-images.githubusercontent.com/8683947/81381801-40429680-9148-11ea-9615-bbdce8d1c87f.gif)

### My environment

vim-xtabline: latest
neovim: NVIM v0.5.0-497-gf04a9a2c9
OS: ArchLinux
xtabline's mode: buffer

```vim
  let g:xtabline_settings = {}
  let g:xtabline_settings.tabline_modes = ['buffers', 'tabs', 'arglist']
  let g:xtabline_settings.buffer_filtering = 0
  let g:xtabline_settings.relative_paths = 0
  let g:xtabline_settings.use_tab_cwd = 0
  let g:xtabline_settings.theme = 'codedark'
  let g:xtabline_settings.map_prefix = 'm'
  " let g:xtabline_settings.enable_mappings = 1
  let g:xtabline_settings.bufline_format = ' N I< l +'
  let g:xtabline_settings.recent_buffers = 99
  let g:xtabline_settings.tab_number_in_buffers_mode = 0

```

**Comment by yutakatay** [comment]
This issue was not reproduced in vim. So it looks like Neovim's problem. Close once. 
I'm sorry.

**Comment by mg979** [comment]
If it's a neovim issue I should fix it anyway. Do you use neovim?

**Comment by yutakatay** [comment]
Thank you for your response.

It seems that it was caused by a compatibility issue between Neovim and coc.nvim. I updated coc.nvim to master and it no longer happens.

It's not a completely xtabline problem. Excuse me :bow: I was wondering if this plugin was the cause because it contained functions such as the Scratch buffer.
https://github.com/mg979/vim-xtabline/blob/158cb91721395dd96326e23e6cd17d4df6985e87/autoload/xtabline/render.vim#L144-L155
